src/components/socialgraph.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { Grid } from "@mui/material";
+
+require('react-dom');
```

Comment:
nit: why are you using cjs require as opposed to ES6 import?

Reply:
Just copy-pasted the debugging code from stackoverflow :p to solve the issue.